com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -452,9 +455,32 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
 
             requiresDepthPrepass |= m_DepthPrimingMode == DepthPrimingMode.Forced;
 
+            // If possible try to merge the opaque and skybox passes instead of splitting them when "Depth Texture" is required.
+            // This optimization is temporarily disabled on VR/XR because of a depth copy y-flip issue on DX11.
+#if ENABLE_VR && ENABLE_XR_MODULE
+            m_CopyDepthPass.renderPassEvent = (!requiresDepthTexture && (applyPostProcessing || isSceneViewCamera || isGizmosEnabled)) ? RenderPassEvent.AfterRenderingTransparents : RenderPassEvent.AfterRenderingOpaques;
+#else
```

Comment:
Do you intend to use `if (cameraData.xr.enabled)` here ?

Reply:
good point, added that check and changed the order of the define checks in https://github.com/Unity-Technologies/Graphics/pull/4628/commits/5f9715edc04a11e98fb76ca2149cf269e112f230